.rubocop.yml

```diff
@@ -1,5 +1,15 @@
-AllCops:
-  TargetRubyVersion: 2.3
-Style/ConditionalAssignment:
-  Exclude:
-    - 'config/initializers/instance_name.rb'
+inherit_gem:
+  rubocop-govuk:
+    - config/default.yml
+    - config/rails.yml
+
+# The suggested fix doesn't work for app/mailers/notifier.rb:
+# `NameError: uninitialized constant ApplicationMailer`.
+Rails/ApplicationMailer:
```

Comment:
We should create an empty class similar to https://github.com/alphagov/support-api/blob/master/app/mailers/application_mailer.rb to solve this linting error.

```rb
class ApplicationMailer < ActionMailer::Base
end
```

and then change https://github.com/alphagov/short-url-manager/blob/241ede7e6ef44dd72751128dadc155a6558f8173/app/mailers/notifier.rb#L1 to inherit from it.

Reply:
D'oh. Yes.

Reply:
I was hoping we could kick this can down the road and fix it when we make Short URL Manager use Notify... but I'll fix it now as it looks easy.